orchagent/orchdaemon.cpp

```diff
@@ -37,6 +37,8 @@ bool OrchDaemon::init()
 {
     SWSS_LOG_ENTER();
 
+    string platform = getenv("platform") ? getenv("platform") : "";
```

Comment:
>getenv [](start = 22, length = 6)

Could you getenv once?
There are other similar problems in this repo, please also check.